### PR TITLE
PR 532

### PR DESCRIPTION
**Pull request by iartemiev**
Fixed errors in:
https://github.com/aws-amplify/docs/pull/532

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: js/analytics.md [hunk 1]
```diff
@@ -433,6 +432,81 @@ Analytics.record({
...
+## Using Amazon Kinesis Firehose
+
+The Amazon Kinesis Firehose analytics provider allows you to send analytics data to an [Amazon Kinesis Firehose](https://aws.amazon.com/kinesis/data-firehose) stream for reliably storing data.
+
+### Installation and Configuration
+
+Register the *AWSKinesisFirehoseProvider* with the Analytics category: 
+
+```javascript
+import { Analytics, AWSKinesisFirehoseProvider } from 'aws-amplify';
```
Comment (sammartinez) [inline js/analytics.md]: Is there any `Amplify.configure(...)` that would need to happen here? Or at least a `Analytics.configure(...)`?

Comment (iartemiev) [reply js/analytics.md]: The configure steps are here: https://github.com/aws-amplify/docs/pull/1021/files#diff-fec60df6a127d419dccaa0fd86ebb178R475-R493

It follows the same order as the Kinesis docs

File: js/analytics.md [hunk 2]
```diff
@@ -433,6 +432,81 @@ Analytics.record({
...
+Analytics.addPluggable(new AWSKinesisProvider());
```
Comment (Amplifiyer) [inline js/analytics.md]: Should this be  `Analytics.addPluggable(new AWSKinesisFirehoseProvider());`?

File: js/analytics.md [hunk 3]
```diff
@@ -433,6 +432,81 @@ Analytics.record({
...
+                "firehose:PutRecordBatch"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+For more information visit [Amazon Kinesis Developer Documentation](https://docs.aws.amazon.com/firehose/latest/dev/controlling-access.html).
+
```
Comment (Amplifiyer) [inline js/analytics.md]: This is probably redundant since the same link is referenced just above it.

**Pull request merged**